src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/ConfigurationTest.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.endpoints.IdamConfigStatusEndpoint;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@IntegrationTest
+public class ConfigurationTest {
+
+    @Autowired
+    private ApplicationContext context;
+
+    @Test
+    public void should_load_custom_actuator_endpoint_for_jurisdictions() {
```

Comment:
A more better integration test would be to use wire mock but that could be done as a separate PR